cl/beacon/handler/blobs.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/ledgerwatch/erigon/cl/persistence/beacon_indicies"
 )
 
+var blobSidecarSSZLenght = (*cltypes.BlobSidecar)(nil).EncodingSizeSSZ()
```

Comment:
Maybe we could move `blobSidecarSSZLenght` inside of `GetEthV1BeaconBlobSidecars` function.

Reply:
no, this is better